xwiki-platform-core/xwiki-platform-realtime/xwiki-platform-realtime-default/pom.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.xwiki.platform</groupId>
+		<artifactId>xwiki-platform-realtime</artifactId>
+		<version>16.3.0-SNAPSHOT</version>
+	</parent>
+	<artifactId>xwiki-platform-realtime-default</artifactId>
+	<name>XWiki Platform - Realtime - Default</name>
+	<description>XWiki Platform - Realtime - Default</description>
+	<properties>
+		<xwiki.jacoco.instructionRatio>0.00</xwiki.jacoco.instructionRatio>
+	</properties>
+	<dependencies>
+		<dependency>
+			<groupId>${project.groupId}</groupId>
+			<artifactId>xwiki-platform-sheet-api</artifactId>
+			<version>${project.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>${project.groupId}</groupId>
```

Comment:
Note: we usually don't use a variable here, and use `org.xwiki.platform` instead.
That's technically correct, but not aligned with the rest of the codebase.